isb_web/manage.py

```diff
@@ -204,7 +204,11 @@ async def login(request: starlette.requests.Request):
     Initiate OAuth2 login with ORCID
     """
     redirect_uri = request.url_for("auth")
-    return await oauth.orcid.authorize_redirect(request, redirect_uri)
+    # check if login is for annotation purpose, if so add query param 
+    if "annotation" in request.query_params and request.query_params["annotation"]=="true" :
```

Comment:
Using the query parameter, we can distinguish whether we are logging in for throughput annotation purposes.